samples/net/ltd_test/README.rst

```diff
@@ -0,0 +1,131 @@
+.. _http-client-sample:
```

Comment:
Looks like this name is already taken (in samples/net/http_client/README.rst
How about we call this one ltd-http-client-sample (and maybe tweak the title too)